.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!---
+BEFORE YOU SUBMIT please search open/closed issues since someone might have asked the same thing before!
+-->
+
+### Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
```

Comment:
Should we ask developers to label their own PR in this case?

```
### Type of issue

<!---  Bug | Feature Request | Help --->
```

That extra data might help active developers of react-navigation to see if this is a production issue or just a clarification on documentation etc.